### ISSUE TITLE
Installing along with Git

### ISSUE THREAD

**Issue by chriscool** [newren/git-filter-repo#107]
About installation along with Git, INSTALL.md says that git-filter-repo "can be installed in the directory pointed to by `git --exec-path`".

I think it's also true that git-filter-repo.1 and git-filter-repo.html can also be installed in the directories pointed to by `git --man-path` and `git --html-path` respectively. So a simple way to install git-filter-repo along with Git, could be simply something like:

```sh
$ cp -a git-filter-repo git_filter_repo.py $(git --exec-path)
$ cp -a git-filter-repo.1 $(git --man-path)/man1
$ cp -a git-filter-repo.html $(git --html-path)
```

It would be nice if there were these instructions somewhere, perhaps in INSTALL.md, or if there was a Makefile target for that. It could be especially useful for people already developing Git :-)

Thanks for your great work anyway!

**Comment by newren** [comment]
Addressed by commit 34f761734b (INSTALL.md: simplify manual installation instructions, 2020-06-13).  Thanks for the pointers!

**Comment by chriscool** [comment]
Nice, thanks!
